actions/trickle-down-changelog/index.js

```diff
@@ -0,0 +1,119 @@
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+import * as semver from 'semver';
+import * as exec from "@actions/exec"
+import * as fs from "fs/promises";
+
+const octokit = github.getOctokit(core.getInput("token"));
+const simpleSemverRegex = /\d+\.\d+\.\d+(-.*)?/;
+const internalBotEmail = "internal@elementor.com";
+async function main() {
+	const currentRef = github.context.ref.replace("refs/heads/", "");
+
+	// we only care about merges to beta/ga branches
```

Comment:
The change-log should be  to all the versions:
ga/Beta/nain

Reply:
yes, but if you merge it to main, then no need to trickle down, it's already at the lowest state